Documentation/Signoff/Logic-gate-subsystem.md

```diff
@@ -0,0 +1,93 @@
+# Logic Gates Subsystem
+
+##  Function of the System
+The function of this subsystem is to introduce the concept of logic gates through an approachable way for someone with little to no experience using AND,OR and NOT gate. The subsystem will be user-friendly by using switches and a single LED to demonstrate these concepts and allow the user to interact with the inputs. The car will be able to be set into mode through the primary screen with the two modes being AND and OR. The two modes will move the car forward and when the user flip the Not gate switch it will invert the ouput of either mode making the car go backward.
+
+
+## Constraint
+
+| Description | Constraints                                                   | Origin            |
+| --- | --------------------------------------------------------------------- | -----------------|
+| Current regulation | Current through the LED shall be no less than 15 mA [3] | LED Constraint |
+| Voltage regulation | System shall use 5v from the Arduino UNO to operate | Device Constraint |
+|  Explicit  | Switches shall be labeled for a user-friendly interaction  | simplicity Constraint|
+| Cost Limit  |Any components and equipment within this subsystem should not exceed $40 dollars and the subsystems total cost should not exceed $120 to help keep the project budget in the 400-900 range| Conceptual Design|
+
+## Buildable schematic
+![image](https://github.com/user-attachments/assets/cff989bc-b4f0-4335-a2e1-3d9a95cced7f)
```

Comment:
If pin D13 is 5v and GND is 0v, is the LED forward or reversed biased? Which one should it be?